character.h

```diff
@@ -1,41 +1,39 @@
 #pragma once
-
-#include <string.h>
+#include <string>
 #include <iostream>
-#ifndef CHARACTER_H
-#define CHARACTER_H
+
 
 
 //A harcol� karakterek oszt�lya
 class Characters

```

Comment:
Miért van többesszámban az osztály neve?! Ez 1 karakter adatait tárolja. A fájlnévben is egyes szám van, legyen az osztály nevében is!

Reply:
Az én értelmezésemben az osztályból több példányt lehet csinálni - és csinálunk is a main.cpp-ben Characters A, Characters B - tehát az osztály az összes létrehozott karakterre értelmezett ezért a többes szám. De amúgy maga a név egyértelmű, és beszédes nem igazán látom, hogy miért lenne szükséges megváltoztatni.

Reply:
De az `A` és a `B` az mindkettő 1-1 karakter. Nem A karakterekről és B karakterekről van szó. Ha az osztály példányai vannak tárolva mondjuk egy vectorban, akkor annak a vectornak a nevében lehet többesszám, mert több karaktert tárol. De az, hogy "az osztályból több példányt lehet csinálni" az szinte minden osztályra igaz, attól még egy példány típusa Character. A `string`-ből is több példányt hozunk létre, mégse `strings`nek hívják.

Reply:
Hmmm...ebben van valami. Valóban logikusabb ebből a megközelítésből.